.github/workflows/sonar.yml

```diff
@@ -36,16 +36,8 @@ jobs:
         uses: gradle/actions/setup-gradle@v4
         with:
           cache-read-only: ${{ github.ref != 'refs/heads/develop' }}
-      - name: Build Gplay Debug
-        run: ./gradlew :app:assembleGplayDebug $CI_GRADLE_ARG_PROPERTIES
-      - name: Build Fdroid Debug
-        run: ./gradlew :app:assembleFdroidDebug $CI_GRADLE_ARG_PROPERTIES
-      - name: Build Sample
-        run: ./gradlew :samples:minimal:assembleDebug $CI_GRADLE_ARG_PROPERTIES
-      - name: Build library fixtures
-        run: ./gradlew assembleDebug createFullJarDebugTestFixtures $CI_GRADLE_ARG_PROPERTIES
-      - name: Build app fixtures
-        run: ./gradlew :app:createFullJarGplayDebugTestFixtures $CI_GRADLE_ARG_PROPERTIES
+      - name: Build debug code and test fixtures
+        run: ./gradlew assembleDebug createFullJarDebugTestFixtures :app:createFullJarGplayDebugTestFixtures $CI_GRADLE_ARG_PROPERTIES
```

Comment:
I understand that minimal app is built with this?

Reply:
It should, but I'd have to double check with a `--dry-run`.

Reply:
Confirmed, it does.